frontends/web/src/contexts/RatesContext.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+*
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createContext } from 'react';
+import { Fiat } from '../api/account';
+import { BtcUnit } from '../api/coins';
+
+type RatesContextProps = {
+    defaultFiat: Fiat;
```

Comment:
It is usually Fiat, but we also have the option to select 'btc' which is not Fiat just a currency, for that reason I slightly prefer to keep the name `defaultCurrency` instead of `defaultFiat`. But I know we already have this a bit wrong in the Fiat type.

Reply:
Hm, I see your point since BTC is not a fiat. Anyways, I could change this to defaultCurrency, same like selectedFiats to selectedCurrencies.

Reply:
Actually I'll change selectedFiats to `activeCurrencies` as this is what the UI displays anyway:

<img width="1162" alt="Screenshot 2023-12-15 at 17 04 59" src="https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/2631308d-d677-4da6-b352-132b3bca8307">

Reply:
nice thanks 👍